provisioning/atat_provisioning.yaml

```diff
@@ -406,263 +276,118 @@ components:
-        - type: object
-          properties:
-            id:
-              type: string
-              example: "csp-environment-id-123"
-    PortfolioOperatorRole:
-      type: object
-      required:
-        - operator_email
-        - access
-      properties:
-        operator_email:
-          type: string
-          example: "portfolio.admin@mail.mil"
-        access:
-          $ref: '#/components/schemas/PortfolioAccess'
-    ApplicationOperatorRole:
-      type: object
-      required:
-        - operator_email
-        - access
-      properties:
-        operator_email:
-          type: string
-          example: "application.user@mail.mil"
-        access:
-          $ref: '#/components/schemas/ApplicationAccess'
-    EnvironmentOperatorRole:
-      type: object
-      required:
-        - operator_email
-        - access
-      properties:
-        operator_email:
-          type: string
-          example: "environment.user@mail.mil"
-        access:
-          $ref: '#/components/schemas/EnvironmentAccess'
-    PortfolioAccess:
-      description: Operator access level to Portfolio
-      type: string
-      enum:
-        - portfolio_administrator
-    ApplicationAccess:
-      description: Operator access level to Application
-      type: string
-      enum:
-        - administrator
-        - contributor
-        - read_only
-    EnvironmentAccess:
-      description: Operator access level to Environment
-      type: string
-      enum:
-        - administrator
-        - contributor
-        - read_only
-    TimePeriod:
-      type: object
-      properties:
-        start_date:
-          type: string
-          format: date
-          example: "2021-10-01"
-        end_date:
-          type: string
-          format: date
-          example: "2021-12-01"
-    CostRequest:
-      type: object
-      properties:
-        period:
-          $ref: '#/components/schemas/TimePeriod'
-        granularity:
-          type: string
-          enum:
-            - "MONTHLY"
-            - "DAILY"
-        type:
-          type: string
-          enum:
-            - "ACTUAL"
-            - "FORECAST"
-    CostResponse:
-      type: object
+            $ref: '#/components/schemas/CostData'
+    CostData:
       properties:
         total:
           type: string
-          description: Total cost (ACTUAL or FORECAST) for entire time period in USD
+          description: Total cost (ACTUAL or FORECAST) for all included time periods in USD
           example: "12345.67"
         results:
           type: array
           items:
             properties:
-              period:
-                $ref: '#/components/schemas/TimePeriod'
+              month:
+                type: string
+                format: date
+                example: "2022-12"
               value:
                 type: string
                 description: Cost (ACTUAL or FORECAST) for the given time period in USD
```

Comment:
Since TimePeriod was removed, would it be better to refer to _month_ rather than _time period_ at :371 and :383?